services/CollectionService/src/index.php

```diff
@@ -25,40 +22,25 @@
   'twig.path' => __DIR__.'/../templates',
 ));
 
-$app['twig'] = $app->share($app->extend('twig', function($twig, $app) {
-  return $twig;
-}));
 $islandoraResourceServiceProvider = new \Islandora\ResourceService\Provider\ResourceServiceProvider;
 
-$app->register($islandoraResourceServiceProvider);
+//Registers Resource Service and defines current app's path for config context
+$app->register($islandoraResourceServiceProvider, array(
+  'islandora.BasePath' => __DIR__,
+));
 $app->mount("/islandora", $islandoraResourceServiceProvider);
 $app->register(new \Islandora\ResourceService\Provider\UuidServiceProvider(), array(
   'UuidServiceProvider.default_namespace' => $app['config']['islandora']['defaultNamespaceDomainUuuidV5'],
 ));
 
-//$app['uuid'] = $app['islandora.uuid5'](rand());
+//We will use a static uuid5 to fake a real IRI to replace <> during turtle parsing
+$app['uuid5'] = $app->share(function () use ($app) {
```

Comment:
Not sure I understand what you mean? He is generating the fake URI here using the uuid5 function and storing it in a variable on the app. 

We could do it inside the route, but this is good too.